utils/build-windows-toolchain.bat

```diff
@@ -635,11 +656,12 @@ cmake ^
   -D SwiftPM_DIR=%BuildRoot%\14\cmake\modules ^
   -D IndexStoreDB_DIR=%BuildRoot%\15\cmake\modules ^
   -D SwiftCollections_DIR=%BuildRoot%\13\cmake\modules ^
+  -D SwiftSyntax_DIR=%BuildRoot%\16 ^
```

Comment:
If I wasn’t misunderstanding the `CMakeLists`…
```suggestion
  -D SwiftSyntax_DIR=%BuildRoot%\16\cmake ^
```

Reply:
Not any more. This is accounted for in the cmake for LSP